pkg/section/local_module.go

```diff
@@ -38,40 +36,21 @@ func (m *LocalModule) Type() string {
 // Configure configures the module section by finding the module
 // for the current path
 func (m *LocalModule) Configure() error {
-	modPaths, err := findLocalModules()
+	modPath, err := findLocalModule()
 	if err != nil {
 		return err
 	}
-	m.Paths = modPaths
+
+	m.Path = modPath
+
 	return nil
 }
 
-func findLocalModules() ([]string, error) {
-	packages, err := packages.Load(
-		// find the package in the current dir and load its module
-		// NeedFiles so there is some more info in package errors
-		&packages.Config{Mode: packages.NeedModule | packages.NeedFiles},
-		".",
-	)
+func findLocalModule() (string, error) {
+	b, err := os.ReadFile("go.mod")
 	if err != nil {
-		return nil, fmt.Errorf("failed to load local modules: %v", err)
-	}
-
-	uniqueModules := make(map[string]struct{})
-
-	for _, pkg := range packages {
-		if len(pkg.Errors) != 0 {
-			return nil, fmt.Errorf("error reading local packages: %v", pkg.Errors)
-		}
-		if pkg.Module != nil {
-			uniqueModules[pkg.Module.Path] = struct{}{}
-		}
-	}
-
-	modPaths := make([]string, 0, len(uniqueModules))
-	for mod := range uniqueModules {
-		modPaths = append(modPaths, mod)
+		return "", fmt.Errorf("go.mod: %w", err)
```

Comment:
The error could include what we were trying to do with this file that failed

```suggestion
		return "", fmt.Errorf("failed reading go.mod: %w", err)
```

Reply:
Looks reasonable. I kept error wrapping in mind: https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Reply:
> Looks reasonable. I kept error wrapping in mind: https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Ah thanks for the link, yeah this probably could just be `"reading go.mod: %w"`, noted for my future use